rds/sql/dialect/postgreSQL.ts

```diff
@@ -294,6 +304,12 @@ export abstract class AbstractPostgreSqlDialect extends diaImpl.ANSI {
         ${sp.bodyCode}
         $BODY$;
         -- ALTER PROCEDURE ${objDefnName}${argTypes} OWNER TO (TODO: OWNER);
+        INSERT INTO plpgsql_check_result
```

Comment:
@Kunjahamed-P @divyaSkumarCI01188 the `plpgsql_check_result` table name should not be hard-coded. Please use SQL interpolation to dynamically generate the proper name.

Reply:
@shah @Kunjahamed-P 
We will check it and update.

Reply:
@shah 
We have tried SQL interpolation in the plpgsql_check in [postgreSQL.ts](https://github.com/gov-suite/governed-im-transform/blob/master/rds/sql/dialect/postgreSQL.ts#L189) in the dialect folder to dynamically generate the proper name. We tried the following steps for achieving this feature 

1. Included link in deps.ts in governed-im-transform/SQL folder 
`export * as governedImRDSModels from https://denopkg.com/gov-suite/governed-im-rds@v0.9.0/models/mod.ts;`
and added the corresponding import statement in postgreSQL.ts.
2. We have tried to call `models.PlPgSqlCheckResult` in storedFunctionDDL 
, but unable to get the result as we expected. Is this the correct scenario to implement SQL interpolation in the plpgsql_check?